cves/kernel/CVE-2017-18017.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A function with an improper handling of memory allowed remote attackers to cause a denial of service attack 
+  (use-after-free and memory corruption) or possibly have unspecified other impact by leveraging the presence of 
```

Comment:
Seems like just a copy-and-paste of the NVD description. Remember that this is about making a vulnerability understandable to second-year SE majors - so they might know about code stuff, but specifics about the linux kernel and security they won't know. 

So you'll want to explain why the firewall is important, how the memory was mishandled, why denial of service could happen, and what else can happen. 

Dig deeper!

Reply:
Also, feel free to bring up interesting details you found throughout your investigation - like how it was discovered.

Reply:
Could also use some more context about what part of the kernel is this is in

Reply:
Added more details on why memory is effected and the results of it happening.